lib/service_provider.rb

```diff
@@ -9,14 +9,18 @@ def initialize(host)
   end
 
   def metadata
-    ignored_methods = [:metadata, :encrypt_responses?, :encryption_opts]
+    ignored_methods = [:metadata, :encrypt_responses?, :encryption_opts, :issuer]
     metadata_methods = self.class.instance_methods(false) - ignored_methods
 
     @metadata ||= metadata_methods.inject({}) do |hash, method|
       hash.merge(:"#{method}" => send(method))
     end
   end
 
+  def issuer
+    @host
```

Comment:
IMO `@host` should be renamed. It's currently a private variable, w/o the new accessor method, and there's nothing particularly host-ish about the string -- it just needs to be unique. Could be a URN (as it is for many of the development sites) or UUID (as it is for the -dashboard app).